tests/records/test_filesystem.py

```diff
@@ -53,23 +53,12 @@ def test_read_write(self, store, result):
 
     async def test_read_locked_key(self, store, result):
```

Comment:
@desertaxle let me know if this isn't maintaining the spirit of this test and i can revisit

Reply:
I'd say it's mostly in the spirit. One of the things this test was checking was that reads will block until the lock is released. I think it's still worth it to have a test for that, but removing the flake is more important.